README.md

```diff
@@ -5,22 +5,22 @@ Based on [Next-Routes](https://github.com/fridays/next-routes) with these change
 - No support for unnamed routes
 - Route can be added only by name, locale and pattern (and optionally page) or options object
 - `Link` and `Router` generate URLs only by route definition (name + params)
-- URLs are prefixed with locale (ie. /en/about)
 - Routes can have data object (if you generate routes dynamically you can pass custom data there)
-- default RequestHandler set locale and nextRoute properties on request (you can get data with ```req.nextRoute.data```) 
 
 ## How to use
 
 Install:
 
 ```bash
-npm install next-routes-with-locale --save
+yarn @join-com/next-routes-with-locale
 ```
 
 Create `routes.js` inside your project:
 
-```javascript
-const routes = module.exports = require('next-routes-with-locale')({ locale: 'en' })
+```typescript
+import nextRoutes from 'next-routes-with-locale'
```

Comment:
`import nextRoutes from '@join-com/next-routes-with-locale'` 
+ change change package name in `package.json`